artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml

```diff
@@ -0,0 +1,35 @@
+name: Windows.Analysis.EvidenceOfDownload
+description: |
+   Simple artifact to find evidence of user download activity.
+   based on the Zone.Identifier alternate data stream that is created alongside with the file downloaded from the internet or intranet. Zone.Identifier is generated by applications when user saves files to the local file system from differnet security zone.
+   This artifact searches the directory provided for any file with alternate data stream named Zone.Identifier and then lists all files with zoneId = 3 or 4 and calculate the hash value of the file and prints the content of Zone.Identifier alternate stream as it could contain useful info in some cases.
+   
+
+reference:
+  - https://cyberforensicator.com/2018/06/26/where-did-it-come-from-forensic-analysis-of-zone-identifier/
+  - https://www.sans.org/security-resources/posters/windows-forensic-analysis/170/download
+   
+author: M.Soheem @msoheem
+
+# Can be CLIENT, CLIENT_EVENT, SERVER, SERVER_EVENT
+type: CLIENT
+
+parameters:
+   - name: DirectoryPath
+     default: "/C:/Users/dfir/Downloads/"
+
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+
+    query: |
+     SELECT get(item=split(string=FullPath, sep=":Zone.Identifier"),member='0') as DownloadedFilePath,
+            hash(path=get(item=split(string=FullPath, sep=":Zone.Identifier"),member='0')) as FileHash,
+            read_file(filename=FullPath,accessor="ntfs") as ZoneIdentifierContent
+     FROM glob(globs=path_join(components=[DirectoryPath,"**/*:Zone.Identifier"]),accessor="ntfs")

```

Comment:
We might want to also add the URL to this query (from https://docs.google.com/presentation/d/1SUPH_D1GPQ7knIfCbj9wGSPuLTSltXjPV95teCY47LU/edit#slide=id.gc2be8b3c533c783_0 
)

Reply:
Okay I placed the content of the alternate stream as not all downloading apps add the URL for example FireFox does not add the URL. however, I'll try to parse it